src/openapi_python_generator/language_converters/python/common.py

```diff
@@ -24,6 +26,24 @@ def get_use_orjson() -> bool:
     return _use_orjson
 
 
+def set_dict_args(value: List[str]):
+    """
+    Set the value of the global variable _dict_args.
+    :param value: value of the variable
+    """
+    global _dict_args
+    _dict_args = value
+
+
+def get_dict_args() -> List[str]:
+    """
+    Get the value of the global variable _dict_args.
+    :return: value of the variable
+    """
+    global _dict_args
+    return _dict_args
```

Comment:
Could it make sense to use an enum for the dict args here? So in the set_dict_args function, we could parse the strings, convert them to enums? That would make the code completion a bit easiert, and we could validate what is passed.